_data/mu-features.yml

```diff
@@ -1,12 +1,12 @@
 content:
     - title: HTTP
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: REST based microservices, where server-side data are made available through representations of data in JSON.
```

Comment:
Is this true though? There may be Http response types that are not represented in Json. For example an image service that serves binary data over http for the web